test/libyul/yulOptimizerTests/unusedStoreEliminator/tstore.yul

```diff
@@ -0,0 +1,23 @@
+{
+    let x := 5
+    tstore(x, 10)
+    sstore(x, 10)
+    pop(mload(0))
+    tstore(x, 10)
+    sstore(x, 10)
```

Comment:
```suggestion
    sstore(x, 20)
```

I'd change it just differentiate between the first `sstore`. Also, please add a test to `yulOptimizerTests/equalStoreEliminator`.

Reply:
Done.

Reply:
Ah yes, sorry. Fixed.